package-config.cmake

```diff
@@ -45,8 +45,22 @@ set(VERSION_CONFIG "${GENERATED_DIR}/${PROJECT_NAME}ConfigVersion.cmake")
 set(PROJECT_CONFIG "${GENERATED_DIR}/${PROJECT_NAME}Config.cmake")
 set(TARGETS_EXPORT_NAME "${PROJECT_NAME}Targets")
 set(namespace "${PROJECT_NAME}::")
+
+set(_PACKAGE_CONFIG_DEPENDENCIES "" CACHE INTERNAL "")
 ENDMACRO(_SETUP_PROJECT_PACKAGE_INIT)
 
+# ADD_PROJECT_DEPENDENCY
+# ----------------------
+#
+# This is a wrapper around find_package to add correct find_dependency calls in
+# the generated config script. All arguments are passed to find_package
+#
```

Comment:
Can you make the doc in the good format for HTML doc ?

Reply:
Done.

I've added a page about this to the doc and I've also updated the documentation to reflect the addition of `NO_INCLUDE_SYSTEM`